pubkey_cache/Cargo.toml

```diff
@@ -11,7 +11,8 @@ anyhow = { workspace = true }
 bls = { workspace = true }
 dashmap = { workspace = true }
 database = { workspace = true }
-log = { workspace = true }
+logging = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order